### PR TITLE
Suggest edits

### PR DESCRIPTION
**Pull request by salaheldinaz**
**Suggest edits:**

- Update web_accounts_list.json
- Create .gitignore
- Add header image to Readme
- Edit Readme file
- Edit check_os() to add 'posix' as default if failed to detect the operating system
- Add descriptions to functions
- Rename finaloutput function to final_output
- fix uname for known_accounts
- Add threading to speed lookup
- edit error style and text
- Save results to file

### CODE REVIEW

File: web_accounts_list_checker.py [hunk 1]
```diff
@@ -247,16 +190,136 @@ def finaloutput():
...
+    sites_not_found = len(args.site) - len(data['sites'])
+    if sites_not_found:
+        print(' -  WARNING: %d requested sites were not found in the list' % sites_not_found)
+    print(' -  Checking %d sites' % len(data['sites']))
+
+else:
+    print(' -  %s sites found in file.' % len(data['sites']))
+
+# create result file
+result_file = open("result.txt", "w")
```
Comment (mccartney) [inline web_accounts_list_checker.py]: It seems to me there's no need to introduce a (temporary) output file for just collecting the outcomes from different threads.

**Comment by WebBreacher** [comment]
@salaheldinaz - any way you can submit these as multiple PRs? It'd be easier to accept the simple ones while we discuss the more challenging/bigger changes.

**Comment by salaheldinaz** [comment]
@WebBreacher Ok I will check how I can do that

**Comment by salaheldinaz** [comment]
I created new pull requests with two edits:
- Readme.md file
- Threading for check_online_presence.py

The other edits already implemented (by you) in the last version. ;)

File: web_accounts_list_checker.py (hunk 1 above)
Comment (salaheldinaz) [reply web_accounts_list_checker.py]: I saw this TODO 
```
2 - Make it so the script will append comment to the entry and output to file
```
that why I added this feature.

@WebBreacher what do you think? Should I add this or not ?

* I will create a new pull if you agree to add it .

**Pull request closed without merging**